### PR TITLE
Add barcelos module and call it in main.py - Barcelos

### PR DESCRIPTION
**Pull request by barcelos-kevin**
Add barcelos module and call it to the main.py

It contains:
1. My Basic Information
2. Goals
3. Simple to do list program.

### CODE REVIEW

File: ourpackage/barcelos.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+def kevin():
+    while True:
+        os.system('cls')
+        print(Fore.YELLOW + "\nHello everyone! I am Kevin Joseph V. Barcelos")
+        print(Fore.WHITE + "1. Basic Information")
+        print("2. Goals")
+        print("3. To Do List (Mini task :>)")
+        print("0. To go back")
+        menu_choice = int(input("Enter your choice: "))
```
Comment (veslino-marc) [inline ourpackage/barcelos.py:16]: I will add my comment - Veslino

**Review by veslino-marc — CHANGES_REQUESTED** [review]

File: ourpackage/barcelos.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+def kevin():
+    while True:
+        os.system('cls')
+        print(Fore.YELLOW + "\nHello everyone! I am Kevin Joseph V. Barcelos")
+        print(Fore.WHITE + "1. Basic Information")
+        print("2. Goals")
+        print("3. To Do List (Mini task :>)")
+        print("4. Comment from Veslino")
+        print("0. To go back")
```
Comment (consultado-kirby) [inline ourpackage/barcelos.py:15]: I will add my comment. - Consultado

**Review by consultado-kirby — CHANGES_REQUESTED** [review]

File: ourpackage/barcelos.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+def kevin():
+    while True:
+        os.system('cls')
+        print(Fore.YELLOW + "\nHello everyone! I am Kevin Joseph V. Barcelos")
+        print(Fore.WHITE + "1. Basic Information")
+        print("2. Goals")
+        print("3. To Do List (Mini task :>)")
+        print("4. Comment from Veslino")
+        print("5. Comment from Consultado")
+        print("0. To go back")
```
Comment (andaya-gener) [inline ourpackage/barcelos.py:15]: I will add my comment. - Andaya

**Review by andaya-gener — CHANGES_REQUESTED** [review]

File: ourpackage/barcelos.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    while True:
+        os.system('cls')
+        print(Fore.YELLOW + "\nHello everyone! I am Kevin Joseph V. Barcelos")
+        print(Fore.WHITE + "1. Basic Information")
+        print("2. Goals")
+        print("3. To Do List (Mini task :>)")
+        print("4. Comment from Veslino")
+        print("5. Comment from Consultado")
+        print("6. Comment from Andaya")
+        print("0. To go back")
```
Comment (piadozo-edriane) [inline ourpackage/barcelos.py:15]: I want to add my comment to your code - Piadozo

**Review by piadozo-edriane — CHANGES_REQUESTED** [review]

File: ourpackage/barcelos.py (hunk 1 above)
Comment (barcelos-kevin) [reply ourpackage/barcelos.py:16]: You have done it.

File: ourpackage/barcelos.py (hunk 2 above)
Comment (barcelos-kevin) [reply ourpackage/barcelos.py:15]: You have done it.

File: ourpackage/barcelos.py (hunk 3 above)
Comment (barcelos-kevin) [reply ourpackage/barcelos.py:15]: You have done it.

File: ourpackage/barcelos.py (hunk 4 above)
Comment (barcelos-kevin) [reply ourpackage/barcelos.py:15]: You have done it.

**Review by consultado-kirby — APPROVED** [review]

**Review by veslino-marc — APPROVED** [review]

**Review by andaya-gener — APPROVED** [review]

**Review by piadozo-edriane — APPROVED** [review]
Thank you :)

**Pull request merged**